docs/Get started/FAQ.md

```diff
@@ -92,14 +92,12 @@ If a table is not available for your host, Fleet will generally handle things be
 
 ### Linux
 
-Fleet Desktop is supported on Ubuntu and Fedora.
+Fleet Desktop is supported on Ubuntu, Fedora, and Debian.
 
-Fedora and some flavors of Ubuntu (e.g. Kubuntu) require a [gnome extension](https://extensions.gnome.org/extension/615/appindicator-support/) and Google Chrome for Fleet Desktop.
+Fedora hosts require a [GNOME extension](https://extensions.gnome.org/extension/615/appindicator-support/) and Google Chrome for Fleet Desktop.
```

Comment:
@lucasmrod we don't need GNOME for Ubuntu anymore right? Just Fedora.

Reply:
Correct. The GNOME extension is only needed for Fedora and Debian. (The GNOME extension is already installed on Ubuntu.)